src/Root.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  Route,
+  HashRouter as Router,
+  Routes,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { NotFound } from './components/NotFound';
+import { Home } from './components/Home';
+import { PeoplePage } from './components/PeoplePage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          {/* Redirect from '/home' to '/' */}
+          <Route path="/home" element={<Navigate to="/" replace />} />
+
+          <Route index element={<Home />} />
+          <Route path="/people" element={<PeoplePage />} />
+          <Route path="/people/:personName" element={<PeoplePage />} />
```

Comment:
The `PeoplePage` component is used for both `/people` and `/people/:personName` routes. Ensure that the component can handle both cases appropriately, or consider separating the logic if different behavior is needed for each route.